api/v1beta2/network_types.go

```diff
@@ -37,6 +37,9 @@ type NetworkStatus struct {
 	// APIServerELB is the Kubernetes api server load balancer.
 	APIServerELB LoadBalancer `json:"apiServerElb,omitempty"`
 
+	// SecondaryAPIServerELB is the secondary Kubernetes api server load balancer.
+	SecondaryAPIServerELB LoadBalancer `json:"secondaryAPIServerELB,omitempty"`
```

Comment:
Does this need to be present? I assume if we create a resource, we want to track it.

Reply:
Yes, makes sense to me to reflect in status